src/UniStaker.sol

```diff
@@ -609,7 +607,8 @@ contract UniStaker is INotifiableRewardReceiver, Multicall, EIP712, Nonces {
   function notifyRewardAmount(uint256 _amount) external {
     if (!isRewardNotifier[msg.sender]) revert UniStaker__Unauthorized("not notifier", msg.sender);
 
-    // We checkpoint the accumulator without updating the timestamp at which it was updated, because
+    // We checkpoint the accumulator without updating the timestamp at which it was updated,
+    // because
```

Comment:
Weird wrapping here

Reply:
Huh yeah, why did this line get changed at all? Otherwise change looks good 👍

Reply:
thanks for the catch, fixed in c4d3e93